.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+* WordPress-Coding-Standards 1.0.0 or higher
+* PHP_CodeSniffer 3.3.0 or higher
+* PHPUnit 4.x, 5.x, 6.x or 7.x
+* PHPCompatibility 8.0 or higher
+
+### Composer setup
+
+Use Composer to install all the necessary dependencies to write and run unit tests. Running
+
+```sh
+composer install
+```
+
+will install PHP_CodeSniffer, WordPress Coding Standards, PHPUnit, PHPCompatibility and security advisories which ensures that your application doesn't have installed dependencies with known security vulnerabilities.
+
+### Other setups
+
+If you have PHP_CodeSniffer (PHPCS) and/or WordPress Coding Standards (WordPressCS) already installed, for instance as git clones, because you either contribute to these libraries as well or you want to develop for or test WPThemeReview with bleeding edge versions of either PHP_CodeSniffer or WordPress Coding Standards, you need to take some additional steps to make it all work.
+
+First, make sure you also have PHPCompatibility and make sure the `installed_paths` for PHP_CodeSniffer is setup correctly.
+
+#### Method 1
+
+Set up environment variables to point to PHP_CodeSniffer and WordPress Coding Standards:
+
+```sh
+PHPCS_DIR: I:/000_GitHub/PHPCS/PHP_CodeSniffer/
+WPCS_DIR: I:/000_GitHub/PHPCS/WordPressCS/
+```
+
+If you do that, everything should work as expected.
+
+#### Method 2
+
+If you do not want to set up environment variables on your system
+
+* Add a file called `.pathtowpcs` to the project root. The only content in that file should be the full absolute path to your WordPressCS install.
+* Copy the `phpunit.xml.dist` file, rename the copied file `phpunit.xml` and make sure it's in the project root.
+Now add the following to that file, adjusting the paths to reflect those on your system:
+
+```xml
+  <php>
+    <env name="PHPCS_DIR" value="path/to/PHP_CodeSniffer/"/>
+    <env name="WPCS_DIR" value="path/to/WordPressCS/"/>
+  </php>
+```
+
+(and don't remove the existing line within the `<php>` block.)
+
+## Writing and running unit tests
+
+The most iportant thing when writing sniffs intended for the theme review, is to have an ample of examples. This makes writing sniffs a lot easier, because you can test agains the give nexamples.
+
+If you want to run unit tests, and if you ran `composer install`, you can now run `composer run-tests` (or `composer run-script run-tests`), which will run the test suite.
+
+The WordPress Theme Review Coding Standards use the PHP_CodeSniffer native unit test suite for unit testing the sniffs.
+
+Once you've started the tests you will see a similar output to this:
+
+```bash
+> @php ./vendor/phpunit/phpunit/phpunit --filter WPThemeReview ./vendor/squizlabs/php_codesniffer/tests/AllTests.php
+PHPUnit 7.3.5 by Sebastian Bergmann and contributors.
+
+............                                                      12 / 12 (100%)
+
+Tests generated 57 unique error codes; 0 were fixable (0%)
+
+Time: 13.55 seconds, Memory: 64.00MB
+
+OK (12 tests, 0 assertions)
+```
+
```

Comment:
"_If you didn't install PHPCS/WPCS/PHPUnit using Composer, you will need to type the above command in to run the unit tests. Make sure you replace the path to PHPUnit and the path to PHPCS when you do._"